Code/Tools/AssetProcessor/native/utilities/GUIApplicationManager.cpp

```diff
@@ -805,7 +805,8 @@ ApplicationManager::RegistryCheckInstructions GUIApplicationManager::PopupRegist
 
 void GUIApplicationManager::InitSourceControl()
 {
-    QSettings settings;
+    // Look in the editor's settings for the Source Control value
+    QSettings settings(QApplication::organizationName(), QString("O3DE Editor"));
     settings.beginGroup("Settings");
     bool enableSourceControl = settings.value("EnableSourceControl", 1).toBool();
```

Comment:
Just wondering if the AP has its own settings independent of the editor

Reply:
In general it does, but for this it makes sense to share the setting

Reply:
Yes, it does but it makes sense to be the same as the Editor so in this case we use that.